<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecordDialog</class>
 <widget class="QDialog" name="RecordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>View Record</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QGroupBox" name="client_group">
       <property name="title">
        <string>Client</string>
       </property>
       <layout class="QFormLayout">
        <item row="0" column="0">
         <widget class="QLabel">
          <property name="text">
           <string>Process:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="clientProcess">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel">
          <property name="text">
           <string>Thread:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="clientThread">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel">
          <property name="text">
           <string>Session:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="clientSession">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="server_group">
       <property name="title">
        <string>Server</string>
       </property>
       <layout class="QFormLayout">
        <item row="0" column="0">
         <widget class="QLabel">
          <property name="text">
           <string>Process:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="serverProcess">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel">
          <property name="text">
           <string>Thread:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="serverThread">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel">
          <property name="text">
           <string>Session:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="serverSession">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="general_group">
     <property name="title">
      <string>General</string>
     </property>
     <layout class="QFormLayout">
      <item row="0" column="0">
       <widget class="QLabel">
        <property name="text">
         <string>Client Port:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="clientPort">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel">
        <property name="text">
         <string>Service:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="service">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel">
        <property name="text">
         <string>Function:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="function">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="cmdbuffer_group">
     <property name="title">
      <string>Command Buffer</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel">
          <property name="text">
           <string>Select:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmdbufSelection">
          <item>
           <property name="text">
            <string>Request Untranslated</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Request Translated</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Reply Untranslated</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Reply Translated</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="cmdbuf">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>clientProcess</tabstop>
  <tabstop>clientThread</tabstop>
  <tabstop>clientSession</tabstop>
  <tabstop>serverProcess</tabstop>
  <tabstop>serverThread</tabstop>
  <tabstop>serverSession</tabstop>
  <tabstop>clientPort</tabstop>
  <tabstop>service</tabstop>
  <tabstop>function</tabstop>
  <tabstop>cmdbufSelection</tabstop>
  <tabstop>cmdbuf</tabstop>
  <tabstop>okButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
